<ui version="4.0" >
 <class>ImportBOMPDownloadDlg</class>
 <widget class="QWidget" name="ImportBOMPDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QLabel" name="lbDesc" >
     <property name="wordWrap">
	     <bool>true</bool>
     </property>
     <property name="text" >
      <string>The university of BONN uses this data strictly for statistic purposes. Please refer to their privacy agreement.</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_14" >
       <property name="text" >
        <string>Your name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leName" >
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_15" >
       <property name="text" >
        <string comment="e-Mail address of the user">Email:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leMail" >
      </widget>
     </item>
    </layout>
   </item>



   <item>
    <widget class="KTextEdit" name="teLicence" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbUnderstoodLicence" >
     <property name="text" >
      <string>I read, understood and accept this license agreement</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
